<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeReportGenerator</class>
 <widget class="QDialog" name="RangeReportGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>153</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Date Range</string>
   </property>
   <widget class="QDateEdit" name="startDate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>End Date</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="generateReport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="errLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
